src/components/FileUpload.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+/**
+ *This is currently working with image files
+ */
+function FileUpload() {
+  // This stores the current state of the file
+  const [file, setFile] = React.useState("");
```

Comment:
Usually we import `useState` at the top
Change your import statement to `import React, { useState } from "react"`
Then this line can just be `... = useState("");`

Reply:
I don't think there is an issue with being explicit, especially when namespace access is only one level .